src/steps/data_validation/utils/classification_dataset_validator.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from src.steps.data_validation.utils.dataset_validator import DatasetValidator
+
+
+class ClassificationDatasetValidator(DatasetValidator):
+    def validate(self):
+        super().validate()  # Call common validations
+        self.validate_labelmap()
+        self.validate_at_least_one_image_per_class()
+
+    def validate_labelmap(self):
+        for dataset_context in self.dataset_collection:
+            if len(dataset_context.labelmap) < 2:
+                raise ValueError(
+                    f"Labelmap must have at least 2 classes in {dataset_context.dataset_name} dataset"
```

Comment:
I would rephrase to something like `Labelmap for dataset {dataset_context.name} is not valid. A classification labelmap must have at least 2 classes. Current labelmap is {dataset_context.labelmap}`